sdk/test/opentelemetry/sdk/trace/export/console_span_exporter_test.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::SDK::Trace::Export::ConsoleSpanExporter do
+  export = OpenTelemetry::SDK::Trace::Export
+
+  let(:captured_stdout) { StringIO.new }
+  let(:spans)    { [OpenTelemetry::Trace::Span.new, OpenTelemetry::Trace::Span.new] }
+  let(:exporter) { export::ConsoleSpanExporter.new }
+
+  before do
+    @original_stdout = $stdout
+    $stdout = captured_stdout
+  end
+
+  after do
+    $stdout = @original_stdout
+  end
+
+  it 'accepts an Array of Spans as arg to #export and succeeds' do
+    _(exporter.export(spans)).must_equal export::SUCCESS
+  end
+
+  it 'accepts an Enumerable of Spans as arg to #export and succeeds' do
+    enumerable = Struct.new(:span0, :span1).new
+    enumerable.span0 = spans[0]
+    enumerable.span1 = spans[1]
```

Comment:
Can just do this in one line?
```ruby
enumerable = Struct.new(:span0, :span1).new(span[0], span[1])
```

Reply:
I've refactored this file and nearby tests to use your suggestions.  thanks! :heavy_check_mark: